podium-uaa/src/main/resources/config/application.yml

```diff
@@ -85,3 +85,9 @@ podium:
     ribbon:
         # comma separated list of profiles to show a ribbon on the GUI
         displayOnActiveProfiles: dev
+
+uaa:
+    security:
+        maxFailedLoginAttempts:
```

Comment:
@gijskant http://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/context/properties/ConfigurationProperties.html

One way would be to use 'ignoreInvalidFields' when the values argumented (String -> long) cannot be coerced and it should default to the set value in the UaaProperties.java.

This is the reason that the default properties for the Async configuration (which are also defaulted with values in PodiumProperties.java) are also explicitly added as values in the application.yml 

Another way would be to also define the default values in the application.yml for the uaa properties.

Not sure if this is a SpringConfiguration bug or not.